exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,36 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number) {
```

Comment:
Use of if/else statements is perfectly valid, however, because the the expression in the if statement returns a boolean, we can just return the evaluation of the expression. So instead of....
```
if(number < 0) {
    return true;
} else {
    return false;
}
```
we could code...
``` 
return num < 0
```